README.md

```diff
@@ -261,8 +261,8 @@ Production capacities are centralized in the [zones.json](https://github.com/tmr
   - Biomass, Solar & Wind: [IRENA](http://resourceirena.irena.org/gateway/countrySearch/?countryCode=IRL)
 - Israel: [Global Power Plant Database](http://datasets.wri.org/dataset/globalpowerplantdatabase)  
 - Italy
-  - Wind & Solar: [IRENA](http://resourceirena.irena.org/gateway/countrySearch/?countryCode=ITA)
```

Comment:
I think this should stay, at least until all the individual regions have associated capacities